querybook/server/logic/datadoc_collab.py

```diff
@@ -43,6 +46,42 @@ def update_datadoc(doc_id, fields, sid="", session=None):
     return doc_dict
 
 
+@with_session
+@with_github_client
+def restore_data_doc(
+    github_client: GitHubClient,
+    datadoc_id: int,
+    commit_sha: str,
+    commit_message: str,
+    current_user: int,
+    sid="",
+    session=None,
+):
+    assert_can_write(datadoc_id, session=session)
+    verify_data_doc_permission(datadoc_id, session=session)
+
+    commit_datadoc = github_client.get_datadoc_at_commit(commit_sha)
+    restored_datadoc = logic.restore_data_doc_from_commit(
+        datadoc_id, commit_datadoc, commit=True, session=session
+    )
+
+    user = user_logic.get_user_by_id(current_user, session=session)
+    assert user is not None, "User does not exist"
```

Comment:
do we need this check? wouldn't assert_can_write fail if the user does not exist? 
also does the current_user needs to be passed over, looks like assert_can_write can access it directly?